pkg/apis/config/defaults.go

```diff
@@ -85,14 +85,16 @@ func defaultDefaultsConfig() *Defaults {
 	}
 }
 
-func asTriState(key string, target **bool) cm.ParseFunc {
+func asTriState(key string, target **bool, defVal *bool) cm.ParseFunc {
 	return func(data map[string]string) error {
 		if raw, ok := data[key]; ok {
 			switch {
 			case strings.EqualFold(raw, "true"):
 				*target = ptr.Bool(true)
 			case strings.EqualFold(raw, "false"):
 				*target = ptr.Bool(false)
+			default:
+				*target = defVal
```

Comment:
```suggestion
func asTriState(key string, target **bool, defaultValue *bool) cm.ParseFunc {
	return func(data map[string]string) error {
		if raw, ok := data[key]; ok {
			switch {
			case strings.EqualFold(raw, "true"):
				*target = ptr.Bool(true)
			case strings.EqualFold(raw, "false"):
				*target = ptr.Bool(false)
			default:
				*target = defaultValue
```
Nothing else is abbreviated so may as well use long name here, and it's nice to see `defaultValue` in the `default:` block

Reply:
Done.